### PR TITLE
Update more references to __experimental menu endpoints to make them stable

### PR DESCRIPTION
**Pull request by talldan**
## Description
Follow up on https://github.com/WordPress/gutenberg/pull/36368 to catch a few references that were missed. Thanks @spacedmonkey for flagging this!

### CODE REVIEW

File: packages/edit-navigation/src/utils/index.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { normalizePath } from '@wordpress/url';
 
 /**
  * The purpose of this function is to create a middleware that is responsible for preloading menu-related data.
- * It uses data that is returned from the /__experimental/menus endpoint for requests
- * to the /__experimental/menu/<menuId> endpoint, because the data is the same.
+ * It uses data that is returned from the /wp/v2/menus endpoint for requests
+ * to the /wp/v2/menu/<menuId> endpoint, because the data is the same.
```
Comment (Mamaduka) [inline packages/edit-navigation/src/utils/index.js:9]: I think this is the last one. It was hiding in regex 😄 

https://github.com/WordPress/gutenberg/blob/d89d485100cc0bf9d116582ff642a1b7059cc0e5/packages/edit-navigation/src/utils/index.js#L47

Comment (Mamaduka) [reply packages/edit-navigation/src/utils/index.js:9]: Fixes in c12aa8f

**Review by Mamaduka — APPROVED** [review]
Thanks, @talldan.

Changes look good to me.

File: packages/edit-navigation/src/utils/index.js (hunk 1 above)
Comment (talldan) [reply packages/edit-navigation/src/utils/index.js:9]: Good catch, thank you 😄

**Pull request merged**